### ISSUE TITLE
question about the groundtruth for semantic segmentation task

### ISSUE THREAD

**Issue by iacopo97** [timbroed/MUSES#5]
Good afternoon, i want to ask a question about which groundtruth did you used or the training of the network? labelTrainIds.png or labelIds.png?

**Comment by timbroed** [comment]
we use the `labelTrainIds.png` files, as they have continuous ids for training.

Here a more detailed explanation from the ReadMe in the .zip:
 - `labelIds`      semantic labels encoded using `png` images, where pixel values encode labels in Cityscapes IDs format. Please refer to the script [helpers/labels.py](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py) in the [Cityscapes GitHub repository](https://github.com/mcordts/cityscapesScripts) for details on the 19 semantic classes included in our semantic annotations, which coincide with the classes that are included in Cityscapes evaluation.
 - `labelTrainIds` semantic labels encoded using `png` images, where pixel values encode labels in Cityscapes trainIDs format.
 - `labelColor`    semantic labels encoded using `png` images, where pixel values encode labels in Cityscapes color format. Purposed for visualization.

**Comment by timbroed** [comment]
The [helpers/labels.py](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py) files also includes the mapping from LabelIds to TrainIds

**Comment by iacopo97** [comment]
Sorry to bother, thank you very much!! Last question, you use a batch size of 16 for training?

**Comment by timbroed** [comment]
yes, for the single modal semantic segmentation experiments we used a batch size of 16

**Comment by iacopo97** [comment]
What about the multimodal?

**Comment by timbroed** [comment]
For the panoptic segmentation, we used a batch size of 8

**Comment by iacopo97** [comment]
Therefore for multimodal semantic segmentation you used a batch size of 16?

**Comment by timbroed** [comment]
We did not include multimodal semantic segmentation in the MUSES paper, only uni-modal

**Comment by iacopo97** [comment]
Ok, thank you, what is your preprocessing pipeline for Mask2Former?
